assets/css/_diversion_page.scss

```
@@ -188,6 +194,7 @@
 
   div {
     margin-bottom: map-get($spacers, 1);
+    line-height: 1.5rem;
```

[Prefer unitless numbers for line-height values](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#prefer_unitless_numbers_for_line-height_values)
```suggestion
    line-height: 1.5;
```

and maybe add a note that we expect the bootstrap reboot to handle this in the future?
```suggestion
    // Expected to be controlled by bootstrap css "reboot" in the future
    line-height: 1.5;
```